lib/parsers/moov_parser.rb

```diff
@@ -55,17 +58,18 @@ def call(io)
     end
 
     # M4A only contains audio, while MP4 and friends can contain video.
-    fmt = format_from_moov_type(file_type)
-    if fmt == :m4a
+    if format == :m4a
       FormatParser::Audio.new(
-        format: format_from_moov_type(file_type),
+        format: format,
         media_duration_seconds: media_duration_s,
         content_type: MP4_AU_MIME_TYPE,
         intrinsics: atom_tree,
       )
+    elsif format == :cr3
+      cr3(decoder, atom_tree, width, height)
     else
```

Comment:
Again not an issue with your code at all but would this not also run if no format was found? Say if the file_type value is something unknown would we just always default it to a video with format nil as oposed to making it clear we do not handle the file format by returning nil?

Reply:
It looks like we default to `:mov` if the `file_type` value isn't something we recognise, so having it default to a video is appropriate based on that. Whether or not that's a good approach is another thing, and I would probably lean "no" 😅 But we have operated with it that way since 2018, apparently, and probably warrants its own story to sort it out.